extra/man/alacritty.5.scd

```diff
@@ -24,6 +26,13 @@ This section documents the root level of the configuration file.
 	All imports must either be absolute paths starting with _/_, or paths
 	relative to the user's home directory starting with _~/_.
 
+	Example:
+
+		import = [++
+  _"~/.my_fancy_colorscheme.toml"_,++
+  _"/home/user/.linux_extra.toml"_,++
```

Comment:
I think it would be best to encourage proper config file organization. So we should maybe use example directories that start with ~/.config/alacritty/` at least.